packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -172,6 +172,10 @@ export class VSXExtension implements VSXExtensionData, TreeElement {
         return this.getData('version');
     }
 
+    /**
+     * @returns undefined if an extension is built-in (so rating should not be displayed)
+     * @returns null if an extension is published but has a rating of 0
+     **/
     get averageRating(): number | undefined {
```

Comment:
[minor] for consistency with other documentation, we should update the jsdoc to:

```ts
/**
* Get an extension's average rating for display purposes.
* - `undefined` for built-in extensions (ratings should not be displayed).
* - `null` for published extensions that have a rating of 0.
*
* @returns the average rating of an extension if available.
**/
```